python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -139,18 +137,48 @@ def get_best_logdir(self, metric, mode="max"):
         elif mode == "min":
             return df.iloc[df[metric].idxmin()].logdir
 
-    def get_best_info(self, metric, mode="max", flatten=True):
-        """Retrieve the best trial based on the experiment metric.
+
+class ExperimentAnalysis(Analysis):
+    """Analyze results from a Tune experiment.
+
+    Parameters:
+        experiment_path (str): Path to where experiment is located.
```

Comment:
Maybe: "Path to a json file representing an experiment state"

At least we should point out that is has to be the json file, not just a generic path. We could also rename `experiment_path` accordingly. The example below does not use the json file.